src/main/java/itm/oss/splitter/Exporter.java

```diff
@@ -1,12 +1,40 @@
 package itm.oss.splitter;
 
 import java.io.*;
+import java.math.RoundingMode;
 import java.util.ArrayList;
 
 public class Exporter {
 
   public static void writePaymentsCsv(String path, ArrayList<Payment> pays) throws IOException {
     // TODO (Issue 9): write CSV with header: from,to,amount
-    throw new UnsupportedOperationException("writePaymentsCsv() not implemented yet");
+
+    File file = new File(path);
+
+    // Automatically close using try-with-resources
+    try (FileWriter writer = new FileWriter(file)) {
+
+      // CSV header
+      writer.write("from,to,amount");
```

Comment:
<img width="628" height="168" alt="image" src="https://github.com/user-attachments/assets/ec9fd75d-119f-4560-9c31-7c2aa3f8ad87" />


As we can see in the exported result (expense.sample.csv), the header is currently written without a newline.

This might cause **concatenation problems** since many CSV parsers (e.g., `OpenCSV`) expect the header to end **with a line break**. Without it, the first data row can be concatenated to the header in some readers.

Consider writing a newline after the header for better CSV compatibility.

```suggestion
      writer.write("from,to,amount\n"); //add a newline after the header for better CSV compatibility.
```

Reply:
Added \n after the CSV header as suggested. Without it, the first data row appeared directly after the header. Now, exported file correctly starts data on a new line.

Reply:
LGTM. Thank you for your kind response!